docs/k8s/user-guide.mdx

```diff
@@ -464,3 +495,194 @@ spec:
                 port:
                   number: <service-port>
 ```
+
+# TCP and TLS Edges {#tcp-tls-edges}
```

Comment:
I wasn't seeing this in the TOC above.

Reply:
@OfTheDelmer This should be resolved now. Some of the headings were not set correctly.